test/common/common.go

```diff
@@ -220,6 +221,17 @@ func VerifyRegistryCache(parentCtx context.Context, log logr.Logger, shootClient
 	By(fmt.Sprintf("Wait until %s Pod is running", name))
 	ExpectWithOffset(1, framework.WaitUntilPodIsRunning(ctx, log, pod.Name, pod.Namespace, shootClient)).To(Succeed())
 
+	// get the architecture of Node the Pod is running on
+	ExpectWithOffset(1, shootClient.Client().Get(ctx, client.ObjectKeyFromObject(pod), pod)).To(Succeed())
+	node := &corev1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: pod.Spec.NodeName,
+		},
+	}
+	ExpectWithOffset(1, shootClient.Client().Get(ctx, client.ObjectKeyFromObject(node), node)).To(Succeed())
+	arch := node.Status.NodeInfo.Architecture
+	log.Info("Node architecture", "arch", arch)
```

Comment:
Maybe also print the node name here?

```suggestion
	log.Info("Node architecture", "name", node.Name, "arch", arch)
```